recipes/openssh/all/conanfile.py

```diff
@@ -0,0 +1,130 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import cross_building
+from conan.tools.env import VirtualBuildEnv, VirtualRunEnv
+from conan.tools.files import copy, get, rmdir, export_conandata_patches
+from conan.tools.gnu import Autotools, AutotoolsToolchain, AutotoolsDeps
+from conan.tools.layout import basic_layout
+from os.path import join
+
+required_conan_version = ">=1.54.0"
+
+
+class PackageConan(ConanFile):
+    name = "openssh"
+    description = "The OpenSSH (portable) suite of secure connectivity tools"
+    license = "SSH-OpenSSH"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.openssh.com/portable.html"
+    topics = ("security", "cryptography", "login", "keychain", "file-sharing", "ssh")
+    package_type = "application"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "with_libcrypto": [None, "libressl", "openssl"],
+        "with_pam": [None, "openpam"],
```

Comment:
```suggestion
        "with_libcrypto": [False, "libressl", "openssl"],
        "with_pam": [False, "openpam"],
```
Not a fan of `None` as an option value. It makes the output of `self.options.get_safe("xyz")` ambiguous. The convention in other recipes is to use `False` in such cases as well.